pkgs/applications/networking/cluster/kuma/default.nix

```diff
@@ -1,19 +1,39 @@
-{ lib, fetchFromGitHub, buildGoModule }:
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, coredns
+, isFull ? true
+, enableGateway ? false
+, pname ? "kuma"
+, version ? "1.3.1"
+, components ? ["kumactl"]
```

Comment:
Such a construct is error prone and sometimes does weird and unexpected things. It would be easier to know what exactly is happening if you move it into a let in or inline.

Reply:
I just moved it to inside struct. Thank's for advice